### DISCUSSION TITLE
Porting AWS IDMSv2 to earlier facter versions.

### DISCUSSION THREAD

**Discussion by itamarbareket** [puppetlabs/facter#2240] [General]
Hi all!

The AWS IDMSv2 is a super important security feature offered on EC2 instances. 
[FACT-2306](https://tickets.puppetlabs.com/browse/FACT-2306) had recently added facter support for it.

I would very like to port this into facter 2.x and 3.x. Is there a possibility a new patch version for those majors would be released if I port the IDMSv2 support?

**Comment by joshcooper** [comment]
This was added to facter 3.x and 4.x in FACT-2904 and FACT-2306.
